script.js

```diff
@@ -0,0 +1,21 @@
+// Função reutilizável fora do escopo
+function verificarSelecao(opcaoSelecionada, btnEnviar, mensagemErro) {
+    const selecionada = Array.from(opcaoSelecionada).some(opcaoSelecionada => opcaoSelecionada.checked);
+    btnEnviar.disabled = !selecionada; 
+    if (selecionada) {
+        mensagemErro.classList.add('is-hidden');
+    } else {
+        mensagemErro.classList.remove('is-hidden');
+    }
+}
+document.addEventListener('DOMContentLoaded', function () {
+    const btnEnviar = document.getElementById('btnEnviar');
+    const opcaoSelecionada = document.querySelectorAll('input[name="voto"]');
+    const mensagemErro = document.getElementById('mensagemErro');
+    opcaoSelecionada.forEach(radio => {
+        radio.addEventListener('change', function () {
```

Comment:
Assim como foi feito com o _forEach_, é possível botar um _array function_ no _addEventListener_ também:
```javascript
radio.addEventListener('change', () => verificarSelecao(opcaoSelecionada, btnEnviar, mensagemErro));
```
Sempre quando for uma **função anônima** (função sem nome), utilize _arrow function_.

Reply:
check